roadmap.md

```diff
@@ -5,44 +5,44 @@ The CloudEvents Roadmap.
 *Setup*
 
 * Establish governance, contributing guidelines and initial stakeholders.
-* Define design goals for CloudEvents V.1
-* Describe the scope of the specification
+* Define design goals for CloudEvents V.1.
+* Describe the scope of the specification.
 * Draft educational materials that provide context for reading the spec.
 
 *0.1*
 
-* Draft specification that WG members agree *could* provide interoperability
-* Document at least 3 sample events that conform to the specification
-* Github repo is organized to be approachable to a engineers who might want to implement the spec
+* Draft specification that WG members agree *could* provide interoperability.
+* Document at least 3 sample events that conform to the specification.
+* Github repo is organized to be approachable to a engineers who might want to
+implement the spec.
 * Additional materials:
-  * use-cases
-  * placeholder for protocol specification, folder and brief starter doc for HTTP
+  * Use-cases.
+  * placeholder for protocol specification, folder and brief starter doc for
+  HTTP.
+* Finalize logo.
+* Deploy a website that features a simple overview, email list and directs
+visitors to Github.
+
 
 *0.2*
 
-* reference implementation of at least two libraries or supporting tools
-* changes to the spec to facilitate adoption
-* HTTP protocol specification
+* Create a reference implementation of at least two libraries or supporting
```

Comment:
Adding the verb suggests something a bit different than I thought this was.  Is the working group creating these libraries?  I can't remember what we were thinking they would do exactly... conformance test maybe?  send/receive event?

Reply:
This is an interesting question.  Is the wg goal to provide only a spec ? libraries or RI ? TCK ?  How we are going to ensure interoperability of the multiple implementations ?

Reply:
@ac360 want to elaborate on this?

Reply:
To make this real, it's important that we provide reference implementations and tools to both use CloudEvents today and integrate it with the broader ecosystem.

I have no preference ATM whether the SG Working Group creates it or a private org does.  I do believe it MUST be done though.

What if we changed the verb to `include` instead of `create`?

Reply:
Maybe say something like... have at least 2 implementations of the spec that can demonstrate interop?

Reply:
+1

Reply:
+1 to `include` or `have`. that won't be hard to fulfill from outside orgs.

Reply:
@ac360 did you want to update anything based on this thread?